Tasks/DownloadPackageV1/Tests/L0DownloadNpmPackageDownloadError.ts

```diff
@@ -0,0 +1,56 @@
+import tmrm = require("azure-pipelines-task-lib/mock-run");
+import path = require("path");
+import { WebApiMock } from "./helpers/webapimock";
+
+let taskPath = path.join(__dirname, "..", "main.js");
+let outputPath: string = path.join(__dirname, "out", "packageOutput");
+let tempPath: string = path.join(__dirname, "temp");
+let zipLocation: string = path.join(tempPath, "singlePackageName.tgz");
+let tr: tmrm.TaskMockRunner = new tmrm.TaskMockRunner(taskPath);
+
+// Set inputs
+tr.setInput("packageType", "npm");
+tr.setInput("feed", "feedId");
+tr.setInput("view", "viewId");
+tr.setInput("definition", "6f598cbe-a5e2-4f75-aa78-e0fd08301a15");
+tr.setInput("version", "versionId");
+tr.setInput("downloadPath", outputPath);
+tr.setInput("extract", "true");
+tr.setInput("verbosity", "verbose");
+
+// Set variables.
+process.env["AGENT_TEMPDIRECTORY"] = tempPath;
```

Comment:
I'm not super familiar with the way the pipeline task l0 tests are set up to be honest - are these values something all of the l0s have set?

Reply:
I copied this setup from another test case in DownloadPackageV1. To the access token mentioned above, it is already checked in sibling tests, but it is not necessary for the test to finish, so I'll remove that here. But if I remove other env variables, tests are starting to fail.

Reply:
Yeah, you can remove them as long as tests aren't failing... some of these variables are used in the test.